clang/test/Driver/clang-offload-bundler-exclude.c

```diff
@@ -0,0 +1,36 @@
+// This test prepares Archive input for clang-offload-bundler
+// and checks -exclude-target command line option.
+// Option should exclude fat_device_aoco object file.
+
+
+// RUN: echo "DUMMY IR FILE" > %t-device
+// RUN: echo "DUMMY IR2 FILE" > %t-device2
+// RUN: echo "DUMMY AOCO FILE" > %t-aoco
+// RUN: echo "DUMMY HOST FILE" > %t-host
+// RUN: echo "DUMMY HOST2 FILE" > %t-host2
+
+// Wrap and compile objects
+// RUN: clang-offload-wrapper -o=%t-device.bc -host=x86_64-unknown-linux-gnu -target=spir64 -kind=sycl %t-device
+// RUN: clang-offload-wrapper -o=%t-device2.bc -host=x86_64-unknown-linux-gnu -target=spir64 -kind=sycl %t-device2
+// RUN: clang-offload-wrapper -o=%t-aoco.bc -host=x86_64-unknown-linux-gnu -target=spir64 -kind=sycl %t-aoco
+// RUN: clang-offload-wrapper -o=%t-host.bc -host=x86_64-unknown-linux-gnu -target=spir64 -kind=sycl %t-host
+// RUN: clang-offload-wrapper -o=%t-host2.bc -host=x86_64-unknown-linux-gnu -target=spir64 -kind=sycl %t-host2
+
+// RUN: llc -filetype=obj -o %t-device.o %t-device.bc
+// RUN: llc -filetype=obj -o %t-device2.o %t-device2.bc
+// RUN: llc -filetype=obj -o %t-aoco.o %t-aoco.bc
+// RUN: llc -filetype=obj -o %t-host.o %t-host.bc
+// RUN: llc -filetype=obj -o %t-host2.o %t-host2.bc
```

Comment:
Does it make sense to use a pre-built binary/archive and put it in `Inputs/SYCL` and just test the behavior of `clang-offload-bundler`?  It reduces external dependencies on other tools.

Reply:
I can put it into Inputs folder. However, I still have dependency on `cat`, `xargs`, `strings` and `grep` tools in the final line of the test.

Reply:
Added